diff --git a/node_modules/@11ty/eleventy-plugin-syntaxhighlight/src/HighlightPairedShortcode.js b/node_modules/@11ty/eleventy-plugin-syntaxhighlight/src/HighlightPairedShortcode.js
index 5dbd602..3ea76b5 100644
--- a/node_modules/@11ty/eleventy-plugin-syntaxhighlight/src/HighlightPairedShortcode.js
+++ b/node_modules/@11ty/eleventy-plugin-syntaxhighlight/src/HighlightPairedShortcode.js
@@ -20,7 +20,7 @@ module.exports = function (content, language, highlightNumbers, options = {}) {
   }
 
   let group = new HighlightLinesGroup(highlightNumbers);
-  let lines = highlightedContent.split("\n");
+  let lines = highlightedContent.split(/\r?\n/);
   lines = lines.map(function(line, j) {
     if(options.alwaysWrapLineHighlights || highlightNumbers) {
       let lineContent = group.getLineMarkup(j, line);
