diff --git a/node_modules/@jspm/generator/dist/generator-2eaaa40d.js b/node_modules/@jspm/generator/dist/generator-2eaaa40d.js
index b617495..ed7aedd 100644
--- a/node_modules/@jspm/generator/dist/generator-2eaaa40d.js
+++ b/node_modules/@jspm/generator/dist/generator-2eaaa40d.js
@@ -3775,7 +3775,7 @@ function _define_property$2(obj, key, value) {
     return obj;
 }
 function combineSubpaths(traceSubpath) {
-    if (traceSubpath.endsWith('/')) throw new Error('Trailing slash subpaths unsupported');
+    if (traceSubpath.endsWith('/')) return traceSubpath;
     return traceSubpath;
 }
 class TraceMap {
@@ -3819,7 +3819,7 @@ class TraceMap {
         // very likely that the user has some CommonJS dependencies, but this is
         // something that the user has to explicitly enable:
         if (isBuiltinScheme(resolved)) return null;
-        if (resolved.endsWith('/')) throw new JspmError(`Trailing "/" installs not supported installing ${resolved} for ${parentUrl}`);
+        if (resolved.endsWith('/')) return;
         try {
             var entry = await this.resolver.analyze(resolved);
         } catch (e) {
@@ -5065,7 +5065,7 @@ function createFetchOptions(cache = true, fetchOptions = {}) {
                 replacer.remove(existingComment.start, existingComment.end, true);
             }
         }
-        replacer.replace(analysis.map.start, analysis.map.end, (comment ? '<!--' + comment + '-->' + newlineTab : '') + esms + '<script type="importmap">' + (whitespace ? newlineTab : '') + JSON.stringify(map, null, whitespace ? 2 : 0).replace(/\n/g, newlineTab) + (whitespace ? newlineTab : '') + '</script>' + preloads + (analysis.map.newScript ? newlineTab : ''));
+        replacer.replace(analysis.map.start, analysis.map.end, (comment ? '<!--' + comment + '-->' + newlineTab : '') + esms + '<script type="importmap">' + (whitespace ? newlineTab : '') + (JSON.stringify(map, null, whitespace ? 2 : 0)?.replace(/\n/g, newlineTab) ?? '') + (whitespace ? newlineTab : '') + '</script>' + preloads + (analysis.map.newScript ? newlineTab : ''));
         return replacer.source;
     }
     async install(install, mode) {
