diff --git a/node_modules/@web/test-runner/dist/reporter/summaryReporter.js b/node_modules/@web/test-runner/dist/reporter/summaryReporter.js
index 1e35853..b88bee5 100644
--- a/node_modules/@web/test-runner/dist/reporter/summaryReporter.js
+++ b/node_modules/@web/test-runner/dist/reporter/summaryReporter.js
@@ -25,7 +25,14 @@ function summaryReporter(opts) {
         var _a, _b;
         const browserName = (browser === null || browser === void 0 ? void 0 : browser.name) ? ` ${dim(`[${browser.name}]`)}` : '';
         for (const result of (_a = results === null || results === void 0 ? void 0 : results.tests) !== null && _a !== void 0 ? _a : []) {
-            log(logger, result.name, result.passed, result.skipped, prefix, browserName);
+            log(
+              logger,
+              flatten ? `${prefix ?? ''} ${result.name}` : result.name,
+              result.passed,
+              result.skipped,
+              prefix,
+              browserName,
+            );
         }
         for (const suite of (_b = results === null || results === void 0 ? void 0 : results.suites) !== null && _b !== void 0 ? _b : []) {
             logSuite(logger, suite, prefix, browser);
@@ -34,9 +41,9 @@ function summaryReporter(opts) {
     function logSuite(logger, suite, parent, browser) {
         const browserName = (browser === null || browser === void 0 ? void 0 : browser.name) ? ` ${dim(`[${browser.name}]`)}` : '';
         let pref = parent ? `${parent} ` : ' ';
-        if (flatten)
+        if (flatten) {
             pref += `${suite.name}`;
-        else
+        } else
             logger.log(`${pref}${suite.name}${browserName}`);
         logResults(logger, suite, pref, browser);
     }
diff --git a/node_modules/@web/test-runner/src/reporter/summaryReporter.ts b/node_modules/@web/test-runner/src/reporter/summaryReporter.ts
index 7be463b..8c90677 100644
--- a/node_modules/@web/test-runner/src/reporter/summaryReporter.ts
+++ b/node_modules/@web/test-runner/src/reporter/summaryReporter.ts
@@ -51,7 +51,14 @@ export function summaryReporter(opts: Options): Reporter {
   ) {
     const browserName = browser?.name ? ` ${dim(`[${browser.name}]`)}` : '';
     for (const result of results?.tests ?? []) {
-      log(logger, result.name, result.passed, result.skipped, prefix, browserName);
+      log(
+        logger,
+        flatten ? `${prefix ?? ''} ${result.name}` : result.name,
+        result.passed,
+        result.skipped,
+        prefix,
+        browserName,
+      );
     }
 
     for (const suite of results?.suites ?? []) {
