diff --git a/node_modules/cem-plugin-module-file-extensions/index.d.ts b/node_modules/cem-plugin-module-file-extensions/index.d.ts
new file mode 100644
index 0000000..107e0e7
--- /dev/null
+++ b/node_modules/cem-plugin-module-file-extensions/index.d.ts
@@ -0,0 +1,5 @@
+import type { Plugin } from '@custom-elements-manifest/analyzer';
+export declare function moduleFileExtensionsPlugin(opts?: {
+  from?: string|RegExp;
+  to?: string|(Parameters<String['replace']>)[1];
+}): Plugin;
