diff --git a/node_modules/element-internals-polyfill/dist/element-internals.d.ts b/node_modules/element-internals-polyfill/dist/element-internals.d.ts
index c428619..36d542b 100644
--- a/node_modules/element-internals-polyfill/dist/element-internals.d.ts
+++ b/node_modules/element-internals-polyfill/dist/element-internals.d.ts
@@ -64,12 +64,12 @@ export declare class ElementInternals implements IElementInternals {
      *
      * If the field is valid and a message is specified, the method will throw a TypeError.
      */
-    setValidity(validityChanges: Partial<globalThis.ValidityState>, validationMessage?: string, anchor?: HTMLElement): void;
+    setValidity(validityChanges: Partial<ValidityState>, validationMessage?: string, anchor?: HTMLElement): void;
     get shadowRoot(): ShadowRoot | null;
     /** The element's validation message set during a call to ElementInternals.setValidity */
     get validationMessage(): string;
     /** The current validity state of the object */
-    get validity(): globalThis.ValidityState;
+    get validity(): ValidityState;
     /** If true the element will participate in a form's constraint validation. */
     get willValidate(): boolean;
 }
