diff --git a/node_modules/@custom-elements-manifest/analyzer/index.d.ts b/node_modules/@custom-elements-manifest/analyzer/index.d.ts
index 18aeda7..482fe41 100644
--- a/node_modules/@custom-elements-manifest/analyzer/index.d.ts
+++ b/node_modules/@custom-elements-manifest/analyzer/index.d.ts
@@ -101,3 +101,16 @@ export interface Plugin {
    */
   packageLinkPhase?(params: PackageLinkPhaseParams): void;
 }
+
+export interface Config {
+  globs?: string[];
+  exclude?: string[];
+  outdir?: string;
+  plugins?: Plugin[];
+  watch?: boolean;
+  dev?: boolean;
+  litelement?: boolean;
+  fast?: boolean;
+  stencil?: boolean;
+  catalyst?: boolean;
+}
diff --git a/node_modules/@custom-elements-manifest/analyzer/src/utils/ast-helpers.d.ts b/node_modules/@custom-elements-manifest/analyzer/src/utils/ast-helpers.d.ts
new file mode 100644
index 0000000..28bf5db
--- /dev/null
+++ b/node_modules/@custom-elements-manifest/analyzer/src/utils/ast-helpers.d.ts
@@ -0,0 +1,4 @@
+import type { SourceFile, Node } from 'typescript';
+
+export declare function hasIgnoreJSDoc(node: Node): boolean;
+export declare function getDeclarationInFile<T extends Node = Node>(name: string, source: SourceFile): T;
diff --git a/node_modules/@custom-elements-manifest/analyzer/src/utils/index.d.ts b/node_modules/@custom-elements-manifest/analyzer/src/utils/index.d.ts
new file mode 100644
index 0000000..fb34b03
--- /dev/null
+++ b/node_modules/@custom-elements-manifest/analyzer/src/utils/index.d.ts
@@ -0,0 +1 @@
+export function resolveModuleOrPackageSpecifier(moduleDoc: any, context: any, elementClass: string): any;
